lib/compilers/clangcl.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2018, Compiler Explorer Authors
```

Comment:
Copyright header date is outdate

Reply:
oh funny - I just copied from clang.ts I think, didn't even reflect on the year :)